app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUpdate.kt

```diff
@@ -56,7 +56,7 @@ class ContactPatientUpdate(
           .contactPatientInfoLoaded()
     }
 
-    return next(updatedModel)
+    return next(updatedModel, LoadCallResultForAppointment(event.overdueAppointment.get().uuid))
```

Comment:
Overdue appointment here is of type `Optional`, would this cause any issues for us? Are we sure we will have overdue appointment here always?

Reply:
umm good catch. Added a commit for this